docs/_articles/en/einstein/einstein-setup.md

```diff
@@ -0,0 +1,98 @@
+5. Click **Save**.
+   This procedure generates an email inviting the new users into the org.
+
+### Visual Studio Code Version
+
+VS Code releases a new version each month with new features and important bug fixes. You must be on VS Code Version 1.76 or higher to run the Einstein for Developers extension. You can manually check for updates from **Help** **> Check for Updates** on Linux and Windows or **Code > Check for Updates** on macOS.
+
+## Get Started with Einstein for Developers
+
+A developer with access to an org enabled for Einstein for Developers can install the Einstein for Developers extensions in VS Code, connect to org and start using the extension.
+
+## Install Einstein for Developers Extensions
+
+To use Einstein for Developerslocally, first install Salesforce Extensions and Salesforce CLI on your desktop.
+
+<a class="slds-button slds-button_brand landing__header-cta slds-m-vertical--x-large" href="https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode">Install the Salesforce Extensions</a>
+
+Then install Einstein for Developers(beta)from the VS Code marketplace.
+
+<a class="slds-button slds-button_brand landing__header-cta slds-m-vertical--x-large" href="https://marketplace.visualstudio.com/items?itemName=salesforcedx-einstein-gpt">Install Einstein for Developers(beta)</a>
+
+## Connect to an Org
+
+The Einstein for Developers(beta) tool is run in the context of a Salesforce org, in Salesforce DX project. To use this tool:
+
+1. Go to **File** > **Open Folder** in the menu and open an existing Salesforce DX project in VS Code, or create a new one.
+
+2. Run the **SFDX: Authorize an Org** command to connect to the Salesforce sandbox org or scratch org that has Einstein for Developers(beta) enabled.
+
+A sidebar panel opens and the Einstein logo on the activity bar confirms that the extension is installed.
+
+![einstein installed](../../../images/einstein-installed.png)
+
+### Use Einstein for Developers(beta) in a Scratch Org
+
+Einstein for Developers(beta)is only available in scratch org editions that can author Apex:
+
+- Developer Edition
+- Enterprise Edition
+
+Activate Einstein for Developers by turning on an add-on scratch org feature.
+
+Edit and save the `config/project-scratch-def.json` file in your DX project to add the “`EinsteinGPTForDevelopers`” feature to your existing feature list.
+
+For example:
+
+```
+{
+"orgName": "Acme Company",
+"edition": "Developer",
+"features":[“Communities”, “ServiceCloud, "EinsteinGPTForDevelopers"]
+}
+```
+
+## Show Einstein Feedback Console View
+
+Run **Einstein: Show Prompt History** from the Command Palette to open the Feedback console. When opened, you can view a running history of your prompts and associated responses. Use 👍, 👎, and comments for each response to provide us feedback about the quality of the generated code. Your feedback during this beta is key to helping us improve AI model quality and overall product.
```

Comment:
Depending on the thread we have going with Flora, we might update this command to point to one of the other History ones